drivers/hopwa_caper/app/models/hopwa_caper/enrollment.rb

```diff
@@ -0,0 +1,140 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HopwaCaper
+  class Enrollment < GrdaWarehouseBase
+    include SqlHelper
+    self.table_name = 'hopwa_caper_enrollments'
+
+    has_many :hud_reports_universe_members,
+             -> do
+               where(::HudReports::UniverseMember.arel_table[:universe_membership_type].eq('HopwaCaper::Enrollment'))
+             end,
+             inverse_of: :universe_membership,
+             class_name: 'HudReports::UniverseMember',
+             foreign_key: :universe_membership_id
+    has_many :services, class_name: 'HopwaCaper::Service', primary_key: :enrollment_id
+
+    def self.as_report_members
+      all.map do |record|
+        ::HudReports::UniverseMember.new(
+          universe_membership_type: sti_name,
+          universe_membership_id: record.id,
+        )
+      end
+    end
+
+    scope :overlapping_range, ->(start_date:, end_date:) {
+      table = arel_table
+      where(
+        table[:entry_date].lteq(end_date).and(
+          table[:exit_date].gteq(start_date).or(table[:exit_date].eq(nil)),
+        ),
+      )
+    }
+
+    scope :latest_by_personal_id, -> {
```

Comment:
This scope name seems disconnected from the contents.  Maybe just `latest_by_id`?  Or `ordered`?